src/app/exam/exam-problems/exam-problems.component.ts

```diff
@@ -49,7 +49,7 @@ export class ExamProblemsComponent implements OnInit, AfterViewInit {
       parseMarkdown(mdString));
   }
 
-  public getProblemOrder(i: number) {
+  public getProblemOrderChar(i: number) {
     return String.fromCharCode(i + 65);
```

Comment:
why not just return an integer?

Reply:
This function is used to map question order 1, 2, 3, ... to A, B, C, ....

Reply:
Figma reference:
![image](https://user-images.githubusercontent.com/15229692/114215284-01404500-9998-11eb-8ae8-c1466203f3d2.png)